templates/base/server/models.py

```
@@ -0,0 +1,121 @@
+from typing import List, Dict, Any, Optional
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)